src/pages/tag.js

```diff
@@ -98,12 +112,18 @@ export default class TagGridPage extends Component {
     } = this.props;
     const pageSize = query && query.limit ? parseInt(query.limit, 10) : uiStore.pageSize;
     const sortBy = query && query.sortby ? query.sortby : uiStore.sortBy;
+    const description = [{ name: "description", content: shop && shop.description }];
 
     return (
       <Fragment>
         <Helmet
           title={`${tag && tag.name} | ${shop && shop.name}`}
-          meta={[{ name: "description", content: shop && shop.description }]}
+          meta={
+            tag.metafields && tag.metafields.length > 0 ?
+              this.renderHeaderMetatags(tag.metafields).concat(description)
```

Comment:
If I add the following meta field to a tag, I get a duplicated description metatag with the shop description.

```js
        {
            "scope" : "name",
            "key" : "description",
            "value" : "THE OVERRIDEN DESCRIPTION",
            "namespace" : "metatag"
        }
```

Reply:
@mikemurray I updated the `renderHeaderMetatags` method to check for whether a `description` has been added, and if it has not, to add the `this.props.shop.description` there.